openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -4,26 +4,29 @@
 import base64
 import json
 import re
-
+import httpretty
```

Comment:
There should still be a blank line separating built-ins from third-party packages. Also, httpretty should be in lexical order with other third-party imports